biggerquery/workflow.py

```diff
@@ -7,15 +9,164 @@ def __init__(self,
                  schedule_interval=DEFAULT_SCHEDULE_INTERVAL,
                  dt_as_datetime=False,
                  **kwargs):
-        self.definition = definition
+        self.definition = self._parse_definition(definition)
         self.schedule_interval = schedule_interval
         self.dt_as_datetime = dt_as_datetime
         self.kwargs = kwargs
 
     def run(self, runtime):
-        for job in self:
+        for job in self.build_sequential_order():
             job.run(runtime=runtime)
 
-    def __iter__(self):
-        for job in self.definition:
-            yield job
\ No newline at end of file
+    def build_sequential_order(self):
+        return self.definition.sequential_order()
+
+    def call_on_graph_nodes(self, consumer):
+        self.definition.call_on_graph_nodes(consumer)
+
+    def _parse_definition(self, definition):
+        if isinstance(definition, list):
+            return Definition(self._map_to_workflow_jobs(definition))
+        if isinstance(definition, Definition):
+            return definition
+        raise ValueError(f"Invalid argument {definition}")
```

Comment:
We might be compatible with python3.5 for a reason :), I suggest removing 3.6 string formatting

Reply:
Ok, I remember now. It's about the python version on Jupyter machines in GCP(it used to be 3.5, I don't know what's the current version).